charts/kyverno/README.md

```diff
@@ -347,8 +347,8 @@ The chart values are organised per component.
 | features.logging.format | string | `"text"` | Logging format |
 | features.logging.verbosity | int | `2` | Logging verbosity |
 | features.omitEvents.eventTypes | list | `["PolicyApplied","PolicySkipped"]` | Events which should not be emitted (possible values `PolicyViolation`, `PolicyApplied`, `PolicyError`, and `PolicySkipped`) |
-| features.policyExceptions.enabled | bool | `true` | Enables the feature |
-| features.policyExceptions.namespace | string | `""` | Restrict policy exceptions to a single namespace |
+| features.policyExceptions.enabled | bool | `false` | Enables the feature |
```

Comment:
Do we want to turn it off by default? This is going to break existing poliyexception users for sure.